linxiangxi/client/include/client/status_code.h

```diff
@@ -0,0 +1,14 @@
+#ifndef __STATUS_CODE_H__
+#define __STATUS_CODE_H__
+
+typedef enum{
+    LOGIN,
+    LOGOUT,
+}Require_code;
+
+typedef enum{
+    SUCCESS,
+    FAILED,
+}Answer_code;
+
+#endif
```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Could not find a newline character at the end of the file.  [whitespace/ending_newline] [5]